presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +increased heap space usage.                            +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++      ``distributed-joins-enabled``       +               ``Boolean``                +              ``true``              +Use hash distributed joins instead of broadcast joins. +
++                                          +                                          +                                    +Distributed joins require redistributing both tables   +
++                                          +                                          +                                    +using a hash of the join key. This can be slower       +
++                                          +                                          +                                    +(sometimes substantially) than broadcast joins but     +
++                                          +                                          +                                    +allows much larger joins. Broadcast joins require that +
++                                          +                                          +                                    +the tables on the right side of the join fit in memory +
++                                          +                                          +                                    +on each machine whereas distributed joins only need to +
++                                          +                                          +                                    +fit in distributed memory across all machines. This    +
++                                          +                                          +                                    +can also be specified on a per-query basis using the   +
++                                          +                                          +                                    +``distributed_join`` session property.                 +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++   ``node-scheduler.network-topology``    +   ``String`` (``legacy`` or ``flat``)    +             ``legacy``             +Sets the network topology to use when scheduling       +
++                                          +                                          +                                    +splits. ``legacy`` will ignore the topology when       +
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
```

Comment:
Good point. Let's keep it generic in description (and possibly give an example of Presto running over HDFS nodes).